test-pom/pom.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2004-2018, Sun Microsystems, Inc., Kohsuke Kawaguchi, CloudBees, Inc.
```

Comment:
Just copy-paste of test/pom.xml. Git fails to capture rename/modification

Reply:
it seems that the `test/pom.xml` still exists (next changeset) so this is not a rename

Reply:
Yes. The most of the content was moved out of there to Parent POM